### ISSUE TITLE
No fresh temp readings after some time

### ISSUE THREAD

**Issue by BerndGaykema** [AdyRock/com.weatherunderground#115]
Hy Andy,

You did this app as well? Nice!
I read my outside temperature (uploaded to WU by my iDrenthe84 weather station) but it happens often that the temperature in Homey stays the same until I restart the app.
I now restart the app 4 times a day (each 6 hours) and that helps.

In my settings I have the number of stations, polling my WU-account, set to 4, although I have just one (Homey). After a restart it works for hours, so I don't think it is the limit which causes the issue.

I tried to connect my sensors directly to Homey Pro 2023 but the 433 MHz receiption of the Homey is rather poor (to be positive).

Hope you know the reason for the lock-up.

Thank you.

**Comment by AdyRock** [comment]
I'm not sure why I wasn't notified of some of these issue by email and as I don't often check I have missed a couple, including this one, so really sorry for the very late response.
As more than a year has past since you posted, has the situation been resolved?

**Comment by BerndGaykema** [comment]
Yes, solved. One of the newer versions solved the issue. No clue which one.
Thank you for replying.
